Assignment1/test/test_distance_formula.py

```diff
@@ -1,11 +1,92 @@
 import pytest
+from pytest import approx
 
 
 @pytest.fixture
 def df():
-    from Assignment1.distance_formula import DistanceFormula
-    return DistanceFormula()
+    from Assignment1.distance_formula import DistanceFormulaCalculator
+    return DistanceFormulaCalculator()
 
 
-def test_hello_world(df):
-    assert df.hello_world() == "hello world"
+def test_distance_is_0(df):
+    assert df.compute_distance(0, 0, 0, 0) == approx(0)
+
+
+def test_distance_is_1_x_axis_only(df):
+    assert df.compute_distance(2, 0, 1, 0) == approx(1)
+
+
+def test_distance_is_1_x_axis_only_neg_val(df):
+    assert df.compute_distance(-2, 0, -1, 0) == approx(1)
+
+
+def test_distance_is_4_x_axis_only_pos_neg(df):
+    assert df.compute_distance(2, 0, -2, 0) == approx(4)
+
+
+def test_distance_is_1_x_axis_only_0_to_1(df):
+    assert df.compute_distance(0, 0, 1, 0) == approx(1)
+
+
+def test_distance_is_1_x_axis_only_1_to_0(df):
+    assert df.compute_distance(1, 0, 0, 0) == approx(1)
+
+
+def test_distance_is_1_x_axis_only_0_to_neg_1(df):
+    assert df.compute_distance(0, 0, -1, 0) == approx(1)
+
+
+def test_distance_is_1_x_axis_only_neg_1_to_0(df):
+    assert df.compute_distance(-1, 0, 0, 0) == approx(1)
+
+
+def test_distance_is_1_y_axis_only(df):
+    assert df.compute_distance(0, 2, 0, 1) == approx(1)
+
+
+def test_distance_is_1_y_axis_only_neg_val(df):
+    assert df.compute_distance(0, -2, 0, -1) == approx(1)
+
+
+def test_distance_is_4_y_axis_only_pos_neg(df):
+    assert df.compute_distance(0, 2, 0, -2) == approx(4)
+
+
+def test_distance_is_1_y_axis_only_0_to_1(df):
+    assert df.compute_distance(0, 0, 0, 1) == approx(1)
+
+
+def test_distance_is_1_y_axis_only_1_to_0(df):
+    assert df.compute_distance(0, 1, 0, 0) == approx(1)
+
+
+def test_distance_is_1_y_axis_only_0_to_neg_1(df):
+    assert df.compute_distance(0, 0, 0, -1) == approx(1)
+
+
+def test_distance_is_1_y_axis_only_neg_1_to_0(df):
+    assert df.compute_distance(0, -1, 0, 0) == approx(1)
+
```

Comment:
The approximations from here up could just be changed to non-approximate values.